src/App.vue

```diff
@@ -9,19 +9,22 @@
     <v-main>
       <router-view></router-view>
     </v-main>
+    <NotificationVue/>
```

Comment:
This should use the kebab case, `notification-vue`. Also, the `Vue` suffix is not necessary and does not confirm to any standard I can find.